src/views/RecIM/components/List/Listing/index.js

```diff
@@ -108,30 +133,48 @@ const ParticipantListing = ({ participant }) => {
         setAvatar(preferredImage || defaultImage);
       }
     };
-    const loadUserInfo = async () => {
-      if (participant.Username) {
-        const profileInfo = await user.getProfileInfo(participant.Username);
-        setName(profileInfo.fullName);
-      }
-    };
-    loadUserInfo();
     loadAvatar();
   }, [participant.Username]);
 
   return (
-    <ListItem key={participant.Username} disableGutters={true}>
-      <Grid container alignItems="center" className={styles.listing}>
-        <ListItemAvatar>
-          <Avatar
-            src={`data:image/jpg;base64,${avatar}`}
-            className={styles.avatar}
-            variant="rounded"
-          ></Avatar>
-        </ListItemAvatar>
-        <Link to={`/profile/${participant.Username}`} className="gc360_link">
-          <ListItemText primary={name} />
-        </Link>
-      </Grid>
+    // first ListItem is used only for paddings/margins
+    // second ListItem (nested inside) is used to layout avatar and secondaryAction
+    <ListItem key={participant.Username}>
+      <ListItem
+        secondaryAction={
+          minimal ? (
```

Comment:
we might want a variable with better naming, I'm not sure what minimal means

Reply:
I tend to agree with this, I think cam was the one who put minimal in. In reality, I think that this is to identify whether the participant listing is for the quick search add, or the team/match pages.